bridge/engine/EngineStreamDirector.h

```diff
@@ -659,8 +683,10 @@ class EngineStreamDirector
         _slidesBitrateKbps = bwKbps;
     }
 
+    bool isNecessaryEstimateSlidesBandwidth() const { return _slidesSsrc != 0 && _slidesBitrateKbps == 0; }
```

Comment:
bool needSlidesBitrateAllocation

then it reads when using
if (needsSlidesBitrateAllocation())
{
    measure();
    allocateBitrate();
}